tests/config/cratedb.ini

```diff
@@ -0,0 +1,15 @@
+[Datasource]
+Name = cratedb
+LogicalQueryFormat = simple_lower
+SchemaName = doc
+CommandLineOverride = -DConnectPluginsPath= FILL ME WITH cratedb_jdbc PATH

```

Comment:
Is there some documentation fragment referring to this file, and the `CommandLineOverride` / `-DConnectPluginsPath` settings, about filling this in?

Reply:
Yeah, in `Frequently found issues and troubleshooting`  https://tableau.github.io/connector-plugin-sdk/docs/tdvt

I just wanted to get the connector out ASAP, the `tests` is only relevant for us, and is currently as you see, not documented, I will definitely add more info some time soon.

Reply:
Thanks for clarifying that this section is WIP. Please go ahead, merge the patch, and good luck with all future improvements. Loving them.